developer/docs/arvo/http.md

```diff
@@ -7,81 +7,99 @@ title: HTTP requests and timers
+              ==                                        ::  40 
+          ==                                            ::  41 
+      +>.$(on &, in-progress &)                         ::  42 
+  ==                                                    ::  43 
+++  sigh-httr                                           ::  44 
+  |=  [wir=wire code=@ud headers=mess body=(unit octs)] ::  45 
+  ~&  'arrive here'                                     ::  46 
+  ^-  [(list move) _+>.$]                               ::  47 
+  ?:  &((gte code 200) (lth code 300))                  ::  48 
+    ~&  [%all-is-well code]                             ::  49 
+    :_  +>.$                                            ::  50 
+    [ost.bow %wait /timer (add ~s10 now.bow)]~          ::  51 
+  ~&  [%we-have-a-problem code]                         ::  52 
+  ~&  [%headers headers]                                ::  53 
+  ~&  [%body body]                                      ::  54 
+  :_  +>.$                                              ::  55 
+  [ost.bow %wait /timer (add ~s10 now.bow)]~            ::  56 
+++  wake-timer                                          ::  57 
+  |=  [wir=wire $~]  ^-  (quip move +>)                 ::  58 
+  ?:  on                                                ::  59 
+    :_  +>.$                                            ::  60 
+    :~  :*  ost.bow                                     ::  61 
+            %hiss                                       ::  62 
+            /request                                    ::  63 
+            ~                                           ::  64 
+            %httr                                       ::  65 
+            %purl                                       ::  66 
+            (need (de-purl:html target))                ::  67 
+        ==                                              ::  68
+    ==                                                  ::  69 
+  [~ +>.$(in-progress |)]                               ::  70 
+::                                                      ::  71 
+++  prep  ~&  target  _`.                               ::  72 
+--                                                      ::  73
 ```
 
-There's some fancy stuff going on here.  We'll go through it line by
-line, though.
+There's some fancy stuff going on here. We'll go through it line by line, 
+though.
 
-There's two kinds of cards we're sending to arvo. A `%hiss` move tells
-`%eyre` to make an HTTP request. It expects a `(unit iden)`, which will
-be null here because we aren't doing any authentication; a mark, in this
-case the http request mark `&httr`; and some data in the form of a
-`cage`, which is a `{mark vase}`. This is confusing, so the mold in the
-code quoted above includes a bunch of otherwise unnecessary faces for
-the purpose of illustration. Compare: `{$hiss p/(unit iden) q/mark
-r/cage}` from `zuse`.
+There's two kinds of cards we're sending to arvo. A `%hiss` move tells `%eyre` 
```

Comment:
Just a PSA, I'm going to try and normalize everywhere stopping the usage of `%vane` instead of Vane. It gets annoying when comparing to other symbol values in the same context, like right here.

I kind of want to do the same for `:app` instead of App (I've been doing that mostly everywhere with Talk and Dojo, because why not, and we'll have new Talk soon that allows capital letters), but that might be a slippery slope since we also have `+generators`, `/+libraries` and `/-surfaces`.